cli.js

```diff
@@ -19,6 +20,7 @@ const cli = meow(`
 	Examples
 	  $ wallpaper unicorn.jpg
 	  $ wallpaper https://octodex.github.com/images/dojocat.jpg
+		$ wallpaper '#FAFAFA'
```

Comment:
Incorrect indentation.

You also need to update it in the readme.

Reply:
Whoops, updated.